app/subscriber/src/features/settings/MyMinisterSettings.tsx

```diff
@@ -17,15 +17,18 @@ export const MyMinisterSettings: React.FC = () => {
     .filter((m) => m.isEnabled)
     .sort((a, b) => a.sortOrder - b.sortOrder || a.name.localeCompare(b.name));
 
-  const handleSubmit = async (values: number[]) => {
+  const handleSubmit = async (values: number[], userInfo: IUserInfoModel) => {
     try {
-      const user = {
+      var user = {
         ...userInfo,
-        preferences: { ...userInfo?.preferences, myMinisters: values },
+        preferences: { ...userInfo.preferences, myMinisters: values },
```

Comment:
This is why javascript is horrible.  Don't do this kind of fake merge.  We need a proper endpoint that updates the user preferences.